code/player/TTTPlayer.cs

```diff
@@ -109,6 +109,11 @@ public override void Respawn()
                 MakeSpectator(false);
             }
 
+            if (Controller is DefaultWalkController defaultWalkController)
+            {
+                defaultWalkController.Breath = DefaultWalkController.MAX_UNDERWATER_BREATH_TIME;
+            }
+
```

Comment:
Can we do this for the walk controller?
https://github.com/TTTReborn/tttreborn/pull/221/files#L27-L31